public/editor/scripts/ui/publisher.js

```diff
@@ -319,6 +321,7 @@ Publisher.prototype.updateDialog = function(publishUrl, allowUnpublish) {
   // "publish"/"cancel" buttons
   if (allowUnpublish) {
     this.dialog.buttons.parent.addClass("hide");
+    this.dialog.publishHeader.text(TEXT_PUBLISHED_HEADER);
```

Comment:
you also need to set the title in the else condition back to `TEXT_PUBLISH_HEADER`